sorter.py

```diff
@@ -15,7 +15,6 @@ def sort_image(pixels, intervals, randomness, sorting_function):
             else:
                 row += interval
             x_min = x_max
-        row.append(pixels[y][0])
```

Comment:
I had to remove this line to avoid the case where an entire row had no intervals/pixels to be sorted. However, I also never figured out the original intention of this line either. Would love if @CasperKoning could shine some light on this for me.